pwa/src/service-worker/set-up-service-worker.js

```diff
@@ -60,11 +59,8 @@ export function setUpServiceWorker() {
     // In development, static assets are handled by 'network first' strategy
     // and will be kept up-to-date.
     if (PRODUCTION_ENV) {
-        // Precache all of the assets generated by your build process.
-        // Their URLs are injected into the manifest variable below.
-        // This variable must be present somewhere in your service worker file,
-        // even if you decide not to use precaching. See https://cra.link/PWA.
-        // Includes all built assets and index.html
+        // Injection point for the precache manifest from workbox-build,
+        // a manifest of app assets to fetch and cache upon SW installation
```

Comment:
A few things are cleaned up in this file now that our own compilation script is handling service workers instead of CRA's